project_proposals/RCM OSS.md

```diff
@@ -0,0 +1,127 @@
+### Where is the code hosted?
+* [Viewer (Freeware)](https://github.com/SchweizerischeBundesbahnen/rcm-dx/releases)
+* [RCM DX Format Specification](https://github.com/SchweizerischeBundesbahnen/rcm-dx)
+* [Example Data Track (100 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TGMS_v003_created-20220429-065432.rcmdx)
+* [Example Data Position (100 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TLS_v001_created-20220429-065432.rcmdx)
+* [Example Data Track Inspection (167 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TSIMS-RAW-COMP_v001_created-20220429-065432.rcmdx)
+* [Example Data Video (937 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TVS_v001_created-20220429-065432.rcmdx)
+##
+### What is the project's main license?
+Eclipse Public License 2.0
+##
+### What other licenses does the project use, e.g. for included 3rd party code?
+Code is not Open Source:
+* BSD 3
+* BSD
+* GPL v3
+* Apache License 1.1 and 2.0
+* LGPLv3 with  static linking  exception
+* MPL v2
+* ISC
+* MIT
+* LGPL V2.1+
+* NUnit License
+* The BSD 2 -  Clause License
+* New BSD License (BSD)
+* LGPL
+* http:// x264licensing.co m/ Professional License
+* GNU LESSER GENERAL  PUBLIC  LICENSE
+* BSD 2
+* The BSD 2 - Clause License
+* Extended WPF Toolkit - Microsoft Public License
+* HDF 5 BSD-style open  source license
+* Reactive  Extensions - Microsoft  Software License Term
+* HDF5DotNet - HDF5  (Hierarchical  Data Format 5)  Software Library  and Utilities
+* Micorsoft .Net  Library License
+* ReactiveUI - Microsoft Public  License
+* [CefSharp](https://raw.githubusercontent.com/cefsharp/CefSharp/master/LICENSE)
+* Oracle  Technology Network License  Agreement
+* MS-PL
+* Eclipse  Public  License
+* CDDL  Version 1.1  or GPL with classpath
+* BSD-style  open  source  license
+* Eclipse  Distribution License  (New-style  BSD  License)
+* The  PostgreSQL License (MIT-, BSD- like)
+* BSD 2- Clause Simplified License
+* BSD-3 License
+* [Visual  Studio  Image  Library](http://download.microsoft.com/download/0/6/0/0607D8EA-9BB7-440B-A36A-A24EB8C9C67E/Visual%20Studio%202013%20Image%20Library%20EULA.docx)
+* aopalliance:ao palliance - Public  Domain
+* New BSD License
+* Eclipse Public License 1.0
+* Proj4 - «MIT style» license
+* Domain  Time II SDK - Commerical
+* Domain Time II  Client - Commerical
+* Telerik UI for WPF - Commercial, royality-free
+* Domain  Time II SDK - Commerical
+* Domain Time II  Client - Commerical
+* SciChart - Commercial (Professional  License)
+##
+### Are any trademarks associated with the project?
+No
+##
+### Does the project have a web site? Where is it?
+[RCM DX Viewer](https://bahninfrastruktur.sbb.ch/de/produkte-dienstleistungen/bahninformatiksysteme/anlagenmanagement/rail-condition-monitoring.html)
```

Comment:
This probably should become a GitHub page under the OpenRail Association. Would that be ok with you?

Reply:
Yes, sure - a start has been made with the readme file: https://github.com/SchweizerischeBundesbahnen/rcm-dx/blob/main/README.md